### PR TITLE
Feature/MTM-38666 update documentation about restricted feature prefix for app name

### PR DESCRIPTION
**Pull request by Airdroplet**
MTM-38666 
* mention application prefix which can't be used while performing operation resulting in creation of new app

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-applications.md [hunk 1]
```diff
@@ -174,6 +174,7 @@ Duplicating an application might be useful if you want to customize a subscribed
 <br><br>
 <img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
 <br><br>
+> **Info:** Prefix 'feature-' is restricted by the platform from being used. You will not be able to create applications using this prefix for application name.
```
Comment (MWindel) [inline content/users-guide/administration-bundle/managing-applications.md:177]:
```suggestion
> **Info:** The prefix 'feature-' is restricted by the platform from being used. You will not be able to create applications using this prefix for the application name.
```

File: content/users-guide/administration-bundle/managing-applications.md [hunk 2]
```diff
@@ -174,6 +174,7 @@ Duplicating an application might be useful if you want to customize a subscribed
 <br><br>
 <img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
 <br><br>
+> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix for the application name.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:177]:
```suggestion
> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name.
```

**Review by kaczynskid-sag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Shall I merge? Is this for 10.14 only?

**Comment by Airdroplet** [comment]
> @Airdroplet Shall I merge? Is this for 10.14 only?

Let's wait a moment for one more approval from UI or BD guys. It should be 10.14 and future releases according to my knowledge.

**Review by c8y-kh4mell — APPROVED** [review]

File: content/users-guide/administration-bundle/managing-applications.md [hunk 3]
```diff
@@ -174,6 +174,7 @@ Duplicating an application might be useful if you want to customize a subscribed
 <br><br>
 <img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
 <br><br>
+> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name.
```
[the remark this answers is not in the document]
Comment (c8y-kh4mell) [reply content/users-guide/administration-bundle/managing-applications.md]: done

**Comment by Airdroplet** [comment]
@BeateRixen please recheck and I think it can be merged if it is fine with you.

File: content/users-guide/administration-bundle/managing-applications.md [hunk 4]
```diff
@@ -174,7 +174,7 @@ Duplicating an application might be useful if you want to customize a subscribed
 <br><br>
 <img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
 <br><br>
-> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name.
+> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name. This also applies to existing applications for use cases when duplicate application feature is used.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:177]:
```suggestion
> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name. This also applies to existing applications in cases where the duplicate application feature is used.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**